www/creating-zooming-images.html

```diff
@@ -45,30 +38,25 @@ <h2>Creating Zooming Images</h2>
         <td><a href="/examples/tilesource-dzi/">DZI</a></td>
     </tr>
     <tr>
-        <td><a href="http://www.maptiler.org/">MapTiler</a></td>
-        <td>desktop app for Windows, Mac, Linux</td>
-        <td><a href="/examples/tilesource-tms/">TMS</a></td>
+        <td><a href="https://github.com/mulka/tiler">Gmap Uploader Tiler</a></td>
+        <td>C++</td>
+        <td><a href="/examples/tilesource-dzi/">DZI</a></td>
     </tr>
     <tr>
-        <td><a href="http://www.vips.ecs.soton.ac.uk/">VIPS</a></td>
-        <td>command line tool and library for a number of languages</td>
-        <td><a href="/examples/tilesource-dzi/">DZI</a> via the <a href="http://libvips.blogspot.co.uk/2013/03/making-deepzoom-zoomify-and-google-maps.html">dzsave</a> feature</td>
+        <td><a href="https://www.microsoft.com/en-us/download/details.aspx?id=52341">HDMake</a></td>
```

Comment:
Looks like this should be "HD View Utilities" and the description should be something like "Photoshop plugin and command-line tool (Windows)".